k2/csrc/properties.cc

```diff
@@ -0,0 +1,31 @@
+#include "fsa.h"
+#include "properties.h"
+
+namespace k2{
```

Comment:
space after k2 (google style guide)

Reply:
@qindazhu 
I would recommend you using https://github.com/google/vim-codefmt

```pip install clang-format```

It will do format for you each time you press `:w`.